core/templates/domain/platform_feature/feature-status-summary.model.spec.ts

```diff
@@ -13,22 +13,15 @@
 // limitations under the License.
 
 /**
- * @fileoverview Unit tests for FeatureStatusSummaryObjectFactory.
+ * @fileoverview Unit tests for feature-status-summary-model.
```

Comment:
Why not ``FeatureStatusSummary`` since that's the name of the object being tested? Ditto elsewhere.

Reply:
done for this and ditto elsewhere

Reply:
Thanks!